charts/vector/templates/ingress.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if .Values.ingress.enabled -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
```

Comment:
Nifty

Reply:
Shamelessly stolen from the "default" resource that's created from `helm create <chart>`